lib/addons/cloud-watch-insights/index.ts

```diff
@@ -1,12 +1,11 @@
 import {Construct} from 'constructs';
-import * as eks from "aws-cdk-lib/aws-eks";
 import * as iam from "aws-cdk-lib/aws-iam";
-import {ClusterInfo} from "../../spi";
-import {conflictsWith, createNamespace, supportsALL} from "../../utils";
+import {ClusterInfo, Values} from "../../spi";
+import {conflictsWith, supportsALL} from "../../utils";
 import {CoreAddOn, CoreAddOnProps} from "../core-addon";
 import {ebsCollectorPolicy} from "./iam-policy";
-import {ManagedPolicy} from "aws-cdk-lib/aws-iam";
-import { KubernetesVersion } from "aws-cdk-lib/aws-eks";
+import {IManagedPolicy, ManagedPolicy} from "aws-cdk-lib/aws-iam";
+import {KubernetesVersion, ServiceAccount} from "aws-cdk-lib/aws-eks";
 
 const versionMap: Map<KubernetesVersion, string> = new Map([
```

Comment:
Is this version map valid for 1.29 and previous releases? if not, let's fix it.

Reply:
It is valid, though I am going to upgrade the default to v1.4 for the newer ones